packages/client/src/runtime/core/errorRendering/applyValidationError.ts

```diff
@@ -469,7 +469,7 @@ function splitPath(path: string[]): [parentPath: string[], fieldName: string] {
 }
 
 function availableOptionsMessage({ green }: Colors) {
-  return `Available options are listed in ${green('green')}.`
+  return `Available options are ` + (green === noop ? `marked with ?` : `listed in ${green('green')}`) + '.'
```

Comment:
Hi @jiashengguo.
Generally, PR looks good. There is one thing I'd like to change: could we adjust the way we check for presence of colors? Green color function being equal to `noop` if colors are disabled is implementation detail that I would like to keep encapsulated.

What I'd suggest instead: add `readonly enabled: boolean` property to `Colors` interface and set it to corresponding values for both `activeColors` and `inactiveColors` implementations. Inside of `availableOptionsMessage`, check `enabled` property instead.

Thank you!

Reply:
sounds good! changed accordingly.